docs/examples/aposmm.rst

```diff
@@ -12,32 +12,26 @@ Optional (see below): petsc4py_, nlopt_, DFO-LS_
 Configuring APOSMM
 ^^^^^^^^^^^^^^^^^^
 
-APOSMM works with a choice of optimizers, some requiring external packages. To
-import the optimization packages (and their dependencies) at a global level
-(recommended), add the following lines in the calling script before importing
-APOSMM::
+APOSMM works with a choice of optimizers, some requiring external packages. Specify
```

Comment:
there are occasions where we cant do this

Reply:
https://github.com/Libensemble/libensemble/pull/1064

Reply:
Should we add a note to point to this issue with PETSc?